src/Organic/SimpleConfigSyncCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
Just pulling out some duplicate code into a helper class. `AdsTxtSyncCommand` is very similar but not _quite_ the same, so I left it alone for now.

Reply:
Awesome 👍